pkgs/development/libraries/arrow-cpp/default.nix

```diff
@@ -216,7 +216,7 @@ stdenv.mkDerivation rec {
     ''
       runHook preInstallCheck
 
-      ctest -L unittest -V \
+      ctest -L unittest \
```

Comment:
Does this affect verbosity of output for failures? Having verbose errors allows to gauge complexity of a build issue at a glance by just the Hydra build logs.

Reply:
Looking at OfBorg logs, they are still very long. I'm not sure what is the effect here.

Reply:
Locally the logs are much shorter. Verbosity is unchanged for failures.